qiskit_optimization/algorithms/slsqp_optimizer.py

```diff
@@ -183,13 +184,17 @@ def solve(self, problem: QuadraticProgram) -> OptimizationResult:
             QiskitOptimizationError: If the problem is incompatible with the optimizer.
         """
         self._verify_compatibility(problem)
+        # we deal with minimization in the optimizer, so turn the problem to minimization
+        max2min = MaximizeToMinimize()
+        original_problem = problem
+        problem = self._convert(problem, max2min)
 
         # construct quadratic objective function
         def _objective(x):
-            return problem.objective.sense.value * problem.objective.evaluate(x)
+            return problem.objective.evaluate(x)
 
         def _objective_gradient(x):
-            return problem.objective.sense.value * problem.objective.evaluate_gradient(x)
+            return problem.objective.evaluate_gradient(x)
```

Comment:
Same here, could be a lambda, I'm not insisting :)

Reply:
We can simply remove `_objective` and `_objective_gradient` by calling `output = fmin_slsqp(problem.objective.evaluate, x_0, ..., fprime=problem.objective.evaluate_gradient, ...)`

Reply:
Agree! Without functions it will look better.

Reply:
It is just passed problem.objective.evaluate and problem.objective.evaluate_gradient now.